dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHIT.java

```diff
@@ -138,6 +138,25 @@ private void toolsIdVersionsVersionIdTypeDescriptorRelativePathNormal() throws E
         assertDescriptor(SUPPORT.getObjectMapper().writeValueAsString(responseObject));
     }
 
+    // Use relative-paths endpoint to return of the test parameter file
+    @Test
+    public void toolsIdVersionsVersionIdTypeDescriptorRelativePathTestFile() throws Exception {
+        Response response = checkedResponse(
+            basePath + "tools/quay.io%2Ftest_org%2Ftest6/versions/fakeName/PLAIN_TEST_CWL_FILE/descriptor/%2Ftest.cwl.json");
```

Comment:
Hmmm, might be a miscommunication on my end.

 I think I'd prefer to use 
`tools/quay.io%2Ftest_org%2Ftest6/versions/fakeName/CWL/descriptor/%2Ftest.cwl.json`

For example, the following lists the files including the test json for CWL
https://dockstore.org:8443/api/ga4gh/v2/tools/quay.io%2Fpancancer%2Fpcawg-sanger-cgp-workflow/versions/2.0.3/CWL/files
then you should be able to get the test file with this (it currently doesn't work)
https://dockstore.org:8443/api/ga4gh/v2/tools/quay.io%2Fpancancer%2Fpcawg-sanger-cgp-workflow/versions/2.0.3/CWL/descriptor/%2Ftest1.json